### ISSUE TITLE
Weeks aren't implemented

### ISSUE THREAD

**Issue by PF94** [mpratt/RelativeTime#12]
This function on ``Lib/RelativeTime/RelativeTime.php`` does not have anything about weeks.

```php
    protected function calculateUnits(DateInterval $interval)
    {
        $units = array_filter(array(
            'years'   => (int) $interval->y,
            'months'  => (int) $interval->m,
            'days'    => (int) $interval->d,
            'hours'   => (int) $interval->h,
            'minutes' => (int) $interval->i,
            'seconds' => (int) $interval->s,
        ));

        if (empty($units)) {
            return array();
        } else if ((int) $this->config['truncate'] > 0) {
            return array_slice($units, 0, (int) $this->config['truncate']);
        }

        return $units;
    }
```

![image](https://user-images.githubusercontent.com/45898787/127376448-a3a5b190-0f56-4d9e-88a1-b78b2c06f086.png)

This doesn't look right in my eyes. Instead of "[number] week(s) ago" being shown, "[number] days ago" is always shown.

Implementing a fix could cause some semi-major breaking changes, but it should be tried.

**Comment by mpratt** [comment]
Hi @chaziz94 
You are indeed right!

I've been looking for an excuse to bump version 1.5 to 2.0 and update this library (using PSR 4, renaming library folders, etc), this might be a good reason to do it. Maybe by adding a new configuration option and changing the behavior it wont cause any backwards compatability issues.

However I have too many things to do at the moment so implementing this is going to take me some time.
I'll be looking into this probably mid August. Feel free to do a PR and I'll look into merging it if you want!

**Comment by mpratt** [comment]
Weeks are implemented on version 1.5.10.
Make sure to set the configuration directive 'use_weeks' => true.

Thanks for the suggestion.
